marklogic-data-hub-central/ui/e2e/cypress/support/pages/load.tsx

```diff
@@ -0,0 +1,230 @@
+     */
+    findByButtonText(text: string) {
+        return cy.findByLabelText(text);
+    }
+
+    //New/Edit modal page objects
+    stepNameInput() {
+        return  cy.findByPlaceholderText('Enter name');
+    }
+
+    stepDescriptionInput() {
+        return cy.findByPlaceholderText('Enter description');
+    }
+
+    selectSourceFormat(format: string) {
+        cy.get('#sourceFormat').click();
+        cy.findAllByText(`${format}`).last().click();
+    }
+
+    selectTargetFormat(format: string) {
+        cy.get('#targetFormat').click();
+        cy.findAllByText(`${format}`).last().click();
+    }
+
+    uriPrefixInput() {
+        return cy.findByPlaceholderText('Enter URI Prefix');
+    }
+
+    cancelButton() {
+        return cy.findByLabelText('Cancel');
+    }
+
+    confirmationOptions(option: string) {
+        return cy.findByLabelText(option);
+    }
+
+    saveButton() {
+        return cy.findByLabelText('Save');
+    }
+
+    //Settings page objects
+    stepNameInSettings() {
+        return cy.get('div p').last();
+    }
+
+    /**
+     * Clicks on a database option
+     * @param db - accepts `STAGING` or `FINAL`
+     */
+    selectTargetDB(db: string) {
+        cy.findByLabelText('targetDatabase-select').click();
+        cy.findByTestId(`targetDbOptions-data-hub-${db}`).click();
+    }
+
+    /**
+     * This input field takes multiple values with special character sequences for keyboard events
+     */
+    targetCollectionInput() {
+        return cy.findByLabelText('additionalColl-select');
+    }
+
+    defaultCollections(collectionName: string) {
+        return cy.findByTestId(`defaultCollections-${collectionName}`);
+    }
+
+    /**
+     * Overwrite the existing default permissions
+     * @param permissions - accepts a comma separated text of roles and capabilities alternately
+     * @example role1,cap1,role2,cap2
+     */
+    setTargetPermissions(permissions: string) {
+        return cy.get('#targetPermissions').clear().type(permissions);
+    }
+
+    /**
+     * Add to the existing default permissions
+     * @param permissions - accepts a comma separated text of roles and capabilities alternately
+     * @example role1,cap1,role2,cap2
+     */
+    appendTargetPermissions(permissions: string) {
+        return cy.get('#targetPermissions').type(`,${permissions}`);
+    }
+
+    selectProvGranularity(options: string) {
+        cy.get('#provGranularity').click();
+        cy.findByTestId(`provOptions-${options}`).click();
+    }
+
+    setBatchSize(batchSize: string) {
+        cy.get('#batchSize').clear().type(batchSize);
+    }
+
+    /**
+     * Textarea that takes a file path in fixtures and pastes the json object {} in the text area
+     * @param fixturePath - file path to headerContent json config file
+     * @see https://docs.cypress.io/api/commands/type.html#Key-Combinations
+     */
+    setHeaderContent(fixturePath: string) {
```

Comment:
more docs for locators that are self explanatory but input format is a need to know